docs/conf.py

```diff
@@ -31,6 +31,7 @@
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
 # ones.
 extensions = [
+    'sphinx.ext.doctest',
```

Comment:
Locally, it doesn't seem to. I've deliberately tried putting errors in and it still builds the docs quite happily without reporting anything. I seem to have to run `make doctest` separately to see any problems